CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.30.1 / 2021-09-27
+
+* [BUGFIX] promtool rules backfill: Prevent creation of data before the start time. #9339
+* [BUGFIX] promtool rules backfill: Do not query after the end time. #9340
+* [BUGFIX] Remote Write: Redact remote write URL when used for metric label. #9383
```

Comment:
This is an ENHANCEMENT since URL is a <string>, which should not contain secrets.